requirements/edx/custom.txt

```diff
@@ -6,7 +6,7 @@
 -e git+https://github.com/edx-solutions/xblock-drag-and-drop.git@92ee2055a16899090a073e1df81e35d5293ad767#egg=xblock-drag-and-drop
 -e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2.git@7b054467159fd2cbe2e0adccf9a0665d36a2a197#egg=xblock-drag-and-drop-v2
 # Temporary new version for A2E course only; installs in parallel with the above version. Aim to remove this and upgrade the above ASAP.
--e git+https://github.com/open-craft/xblock-drag-and-drop-v2.git@a5a2704aca93544aaf17470eb2e997539e4c18ec#egg=xblock-drag-and-drop-v2-new
+-e git+https://github.com/open-craft/xblock-drag-and-drop-v2.git@1167385e492c7a79407db991c68dc3838e62c8ca#egg=xblock-drag-and-drop-v2-new
```

Comment:
@ciuin Should we switch to use edx-solutions fork here? (I'm not sure though, as it was using open-craft since the beginning of time)

Reply:
@e-kolpakov I'm not sure what you mean. The v3-temporary-branch with all the -new nomenclature doesn't exist in the edx-solutions fork. We could copy it, but isn't all that supposed to be going away soon?

Reply:
@e-kolpakov  Nope,`xblock-drag-and-drop-v2-new` and its `v3-temporary-branch` are supposed to stay on the `open-craft` fork. It's just a temporary branch / nasty hack, and we really don't want to be using it long-term, so it only exists on our fork.